python/lsst/validate/drp/matchreduce.py

```diff
@@ -55,7 +55,7 @@ def build_matched_dataset(repo, dataIds, matchRadius=None, safeSnr=50.,
     dataIds : `list` of `dict`
         List of `butler` data IDs of Image catalogs to compare to reference.
         The `calexp` cpixel image is needed for the photometric calibration.
-    matchRadius :  afwGeom.Angle(), optional
+    matchRadius :  geom.Angle(), optional
```

Comment:
I think docstrings may need the full definition and should also be in back ticks:
```
matchRadius : `lsst.geom.Angle`, optional
```